definitions/procedures/services/list.rb

```diff
@@ -0,0 +1,21 @@
+#TODO: figure out why I need this?
+require_relative 'base.rb'
```

Comment:
@iNecas @mbacovsky Is there a better way to do this? I would have thought the base class would be required already here, but it wasn't. Maybe I am missing something obvious  :man_shrugging:

Reply:
@johnpmitsch,

Yes, you don't need to require base class here. Instead of Base class, you just need to inherit List class from ForemanMaintain::Procedure. Example - 

```
module Procedures::Services
  class List < ForemanMaintain::Procedure
       # code
  end
```

Reply:
@kgaikwad 

The [base class](https://github.com/theforeman/foreman_maintain/pull/151/files/7c5cf13a2a0de4fd7c4de0cfb8c96bc9acf0fc1a#diff-003670ad8a995e0ca76650538024b9cb) is for the services  procedures to share methods, and it inherits from `ForemanMaintain::Procedure`. I'm fine with the require_relative here but I am just wondering if there is a better way such as adding something in `lib/foreman_maintain.rb